### DISCUSSION TITLE
Exposing API endpoints for digit system

### DISCUSSION THREAD

**Discussion by vipinvijay1986** [egovernments/Digit-Core#28] [DIGIT Platform]
Does the DIGIT system exposes API endpoints to the public or can we use them for data analysis ? ( Like a Twitter provides API to view the tweets ) .

**Comment by eGovMan** [comment]
DIGIT does expose the APIs. We do need to make it more friendly for developers to register, get approval, download API keys and build solution using the APIs. This is an area where we can start discussion and formalise the requirements. I would call this DIGIT developer portal.

**Reply by vipinvijay1986** [reply]
@eGovMan Does it mean that developer can use API only to enhance existing produce  rather than building own product using API.( For example if I want to do analysis then only way is to enhance dashboard rather than using API ??). 
I have few  more points on this :  
- As DIGIT is deployed by different vendors so will they be part of this? because the data will be hold by specific system. 
- Few examples of usecase can be : 
   1. Licence renewal trend 
   2. Defaulters in system 
   3. Area based trend analysis. 
   4. Integration with different system for payment alerts ( For example Amazon have integration with BESCOM )
